docs/connecting-to-ethereum/setting-up-a-name.md

```diff
@@ -0,0 +1,42 @@
+---
+id: setting-up-a-name
+title: Setting up a name
+---
+
+Setting a name for your org is an easy way to make it more discoverable and human-readable. You can set a name via the Radicle web client, [app.radicle.network][wc].
+
+First, navigate to your org's page. If you are a member/owner of the org, you should should see a button next to the Name field that says **Set**. 
+
+![Org Profile Page][sn1] 
+
+Once you click set, you can input the name that you'd like to set to this org.
+
+![Setting a name][sn2] 
+
+To set a name, you must update the name registration to resolve to the address of the org. To do so, copy the address of the org, and navigate to the name registration page. Update the registration record by adding the org address to the **Address** field. 
```

Comment:
Instead of saying "navigate to the name registration page" say the actual button or link you have to click on to get to this page.

Reply:
Because this is not clear at all to me from the screenshots which page is the name registration page or how to even get to that page.